data/src/main/scala/com/exini/dicom/data/data.scala

```diff
@@ -21,151 +21,141 @@ import java.nio.charset.StandardCharsets
 import java.nio.{ ByteBuffer, ByteOrder }
 import java.time.{ ZoneOffset, ZonedDateTime }
 import java.util.UUID
+import scala.collection.immutable
 
 package object data {
 
+  type Bytes = immutable.ArraySeq[Byte]
```

Comment:
Here we define a new type which resolves to Scala's immutable array representation which according to the docs is efficient and has a small footprint